tests/contrib/elasticsearch/test.py

```diff
@@ -145,6 +145,45 @@ def test_elasticsearch(self):
         es.indices.delete(index=self.ES_INDEX, ignore=[400, 404])
         es.indices.delete(index=self.ES_INDEX, ignore=[400, 404])
 
+    def test_elasticsearch_ot(self):
+        """Shortened OpenTracing version of test_elasticsearch."""
+        from tests.opentracer.utils import init_tracer
```

Comment:
Imports at the top of the module. Also can you manually check it with `flake8`?

Reply:
`flake8` doesn't seem to complain, but I'll move it to the top.